terrain/voxel_terrain.cpp

```diff
@@ -1018,15 +1018,13 @@ void VoxelTerrain::_process() {
 					continue;
 				}
 
+				collidable_surfaces.push_back(data.blocky_surfaces.surfaces[i]);
```

Comment:
Just use `surface`. Also, prefer keeping this after the `is_surface_triangulated(surface)`.